src/containers/AboutPage/AboutPageContent.tsx

```diff
@@ -134,29 +154,44 @@ const AboutPageContent = ({ article: _article, frontpage }: Props) => {
           <HeroContent>
             <HomeBreadcrumb items={crumbs} />
           </HeroContent>
-          <ArticleWrapper>
-            <ArticleHeader>
+        </OneColumn>
+        <ArticleWrapper>
+          <OneColumn>
+            <ArticleHeader padded>
               <Heading id={SKIP_TO_CONTENT_ID} tabIndex={-1}>
                 {article.transformedContent.title}
               </Heading>
               <Text textStyle="body.xlarge">{article.transformedContent.introduction}</Text>
             </ArticleHeader>
-            <ArticleContent>
-              {article.transformedContent.content}
-              <ArticleByline
-                licenseBox={
-                  <LicenseBox
-                    article={article}
-                    copyText={article?.transformedContent?.metaData?.copyText}
-                    oembed={undefined}
-                  />
-                }
-                displayByline={false}
-              />
-            </ArticleContent>
-          </ArticleWrapper>
-        </OneColumn>
-        <AboutPageFooter frontpage={frontpage} />
+            <ArticleContent padded>{article.transformedContent.content}</ArticleContent>
+          </OneColumn>
+          <ArticleFooter>
+            <OneColumn>
+              <ArticlePadding asChild>
+                <AccordionRoot multiple>
+                  <AccordionItem value={"rulesForUse"}>
+                    <AccordionItemTrigger>
+                      <Heading asChild consumeCss textStyle="label.medium" fontWeight="bold">
+                        <h2>{t("article.useContent")}</h2>
+                      </Heading>
+                      <AccordionItemIndicator asChild>
+                        <ArrowDownShortLine />
+                      </AccordionItemIndicator>
+                    </AccordionItemTrigger>
+                    <AccordionItemContent>
+                      <LicenseBox
+                        article={article}
+                        copyText={article?.transformedContent?.metaData?.copyText}
+                        oembed={undefined}
+                      />
+                    </AccordionItemContent>
+                  </AccordionItem>
+                </AccordionRoot>
```

Comment:
Re-implementer deler av byline her for å kvitte oss med de delene av byline vi ikke vil ha. Kanskje greit å gjøre dette til en komponent vi kan ha i frontend-packages. Hva tenker folket?